src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/TaskResourceSpecificationTest.java

```diff
@@ -226,7 +226,7 @@ void should_build_task_query_with_all_parameters() {
             new SearchParameterList(JURISDICTION, SearchOperator.IN, singletonList("IA")),
             new SearchParameterList(STATE, SearchOperator.IN, singletonList("ASSIGNED")),
             new SearchParameterList(LOCATION, SearchOperator.IN, singletonList("location")),
-            new SearchParameterList(CASE_ID, SearchOperator.IN, singletonList("caseId")),
+            new SearchParameterList(CASE_ID, SearchOperator.IN, singletonList("case_id")),
```

Comment:
Probably want to ensure there's a test that covers that the camelCase still works.

Reply:
done